server/db.js

```diff
@@ -0,0 +1,58 @@
+const mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost/abodely', { useNewUrlParser: true, useCreateIndex: true });
+
+const oneReviewSchema = new mongoose.Schema({
+  user_name: { type: String, unique: true },
+  user_photo: String,
+  review_text: String,
+  accuracy: Buffer,
```

Comment:
I'd be interested to know why we chose to use the datatype buffer data type here instead of a Number?

Reply:
Jordan suggested it as a potential challenge instead of using numbers. Since all ratings will be within 1 to 5, buffers will take up only 3 bits instead of 32-bits.